README.md

```diff
@@ -52,9 +52,10 @@ Create a new file locally called `.env.local` and add the following:
 
 ```bash
 WORDPRESS_HOST="[host]"
+PRODUCTION_URL="[productionUrl]
```

Comment:
one consideration, is inside of the app, in order to provide a URL for meta tags with the site URL, i'm actually grabbing the `homepage` property of the `package.json` file

i'm wondering what your thought on that approach is, if you like that, maybe we can use that same variable instead of adding an .env variable, or, if you think having the .env variable makes more sense, we can update the app (in a separate pull request) to use this environment variable

i dont feel strongly one way or another but i like the idea of using the same variable between the 2

https://github.com/colbyfayock/next-wordpress-starter/blob/3ac473a4ddcd02d111e6a8ca1d74433fec6a7de4/src/hooks/use-site.js#L9

https://github.com/colbyfayock/next-wordpress-starter/blob/e3dda779d75a9efabbcfdd70d31bc93e9d912477/src/components/Layout/Layout.js#L40

https://github.com/colbyfayock/next-wordpress-starter/blob/c6740a28a425f7c972959afca17d50406243757d/README.md#packagejson

https://github.com/colbyfayock/next-wordpress-starter/blob/c6740a28a425f7c972959afca17d50406243757d/package.json#L3

Reply:
Oops, I missed that part. Changing it now.